lib/media/mp4_ttml_parser.js

```diff
@@ -31,10 +31,12 @@ goog.require('shaka.util.Mp4Parser');
  * @param {number} offset
  * @param {?number} segmentStartTime
  * @param {?number} segmentEndTime
+ * @param {?boolean} useRelativeCueTimestamps
```

Comment:
"?" in front of the type means "nullable". E. g. segmentEndTime could be a number, or it could be null. Notice, how the offset is just {number}, meaning we always expect it to be a number, passing null won't compile.

In case of useRelativeCueTimestamps, it always should be either true or false, which means we don't need the "?" before "boolean". Same stands for all the other places where it is declared.